pkg/asset/manifests/template.go

```diff
@@ -1,5 +1,31 @@
 package manifests
 
+// AWS holds encoded credentials and is used to generate cloud-creds-secret
+type AWS struct {
```

Comment:
The previous names of these types were better because they offered a better description of what they were. As it is, a type of `AWS` in the `pkg/asset/manifests` package is not very clear.

Reply:
yup, i misunderstood you yesterday, got it